dali/operators/reader/loader/loader.h

```diff
@@ -120,10 +121,10 @@ class Loader {
     if (!loading_flag_) {
       PrepareMetadata();
     }
-    TimeRange tr("[Loader] ReadOne", TimeRange::kGreen1);
+    DomainTimeRange tr("[DALI][Loader] ReadOne", RangeBase::kGreen1);
     // perform an initial buffer fill if it hasn't already happened
     if (!initial_buffer_filled_) {
-      TimeRange tr("[Loader] Filling initial buffer", TimeRange::kBlue1);
+      DomainTimeRange tr("[DALI][Loader] Filling initial buffer", RangeBase::kBlue1);
```

Comment:
Hmm, my only nitpick would be that we use RangeBase instead of DomainTimeRange::kBlue... directly.

Reply:
Done